src/app/public/modules/file-attachment/file-attachment.component.html

```diff
@@ -0,0 +1,69 @@
+<div
+  class="sky-single-file-wrapper">
```

Comment:
This goes along with other comments around class names, but I've noticed we're using `sky-single-file-` and `sky-file-attachment-` prefixes interchangeably. Should we only be using one, or do the two have a different purpose?

Reply:
Bad cleanup on my part again. I think we can get rid of `single` since the component is just called `file-attachment`.